ossfuzz/ossfuzz.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -eu
+
+# This script is called by the oss-fuzz main project when compiling the fuzz
+# targets. This script is regression tested by travisoss.sh.
+
+# Save off the current folder as the build root.
+export BUILD_ROOT=$PWD
+
+# lz4 uses CPPFLAGS rather than CXX flags.
+export CPPFLAGS="${CXXFLAGS}"
```

Comment:
Not sure what is meant here.
The proper "equivalent" to `CXXFLAGS` is `CFLAGS`,
although they are named differently for a reason : `CFLAGS` is for C compiler, and `CXXFLAGS` is for C++ compiler. Although some flags are common, there are also C compiler flags which make no sense for C++, and vice versa.

On the other hand `CPPFLAGS` is about "preprocessor flags", for example the include directories. And this concept is common for both C and C++.

`CPPFLAGS` is implied by default for both C andC++ compilations (but not linking), so it should not be necessary to "repeat" its content inside `CXXFLAGS`.

Reply:
Ah, fair enough. I possibly got the wrong end of the stick here. Will take a look